memebox-streamdeck/public/common.js

```diff
@@ -727,14 +725,7 @@ function connectElgatoStreamDeckSocket(
 
 /* legacy support */
 
-function connectSocket(
-  inPort,
-  inUUID,
-  inMessageType,
-  inApplicationInfo,
-  inActionInfo
-) {
-  connectElgatoStreamDeckSocket(
+function connectElgatoStreamDeckSocket(
```

Comment:
this part needs to reverted - as far I understood this `connectSocket` is called in some streamdeck versions so we can't just remove it fully